x-pack/plugins/security_solution/public/exceptions/routes.tsx

```diff
@@ -29,9 +33,9 @@ const ExceptionsRoutes = () => (
 
 const ExceptionsListDetailRoute = () => (
   <PluginTemplateWrapper>
-    <TrackApplicationView viewId={SecurityPageName.sharedExceptionListDetails}>
+    <TrackApplicationView viewId={SecurityPageName.exceptionListDetails}>
       <ListsDetailView />
-      <SpyRoute pageName={SecurityPageName.sharedExceptionListDetails} />
+      <SpyRoute pageName={SecurityPageName.exceptionListDetails} />
```

Comment:
This pageName should not be used here, since it is a dynamic page, we should keep using `pageName={SecurityPageName.exception}`. 

If we want to generate a trailing breadcrumb with the endpoingList name we could add a state prop:
```
<SpyRoute pageName={SecurityPageName.exceptions} state={{ exceptionListName }}/>
```
(not here, somewhere we have the exceptionList name defined)

And create a trailing breadcrumbs module similar to:

https://github.com/elastic/kibana/blob/ab2eb9db28d10e812e874775c22d43a6723143ac/x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/utils.ts#L46